python/mirage/_cython/core.pyx

```diff
@@ -101,6 +101,110 @@ bfloat16 = dtype('bf16')
 float32 = dtype('fp32')
 float64 = dtype('fp64')
 
+def get_kn_operator_type_string(int op_type):
+    if op_type == 1000:
```

Comment:
Instead of using absolute numbers, can we use `KN_XXX_OP` defined in `CCore.pxd`? This change will make it easier to update the encoding of KN/TB operators (e.g., we may need to change the ids corresponding to each operator type).